articles/quickstart/native/ios-swift-siwa/00-login.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to a Swift application using Sign In With Apple.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ios
+  - swift
+  - siwa
+github:
+  path: 00-Login
+contentType: tutorial
+useCase: quickstart
+requirements:
+  - CocoaPods
+  - Xcode 11 Beta 6
+  - iOS 13 Beta 8 Device
+---
+
+<!-- markdownlint-disable MD002 MD041 -->
+
+## Before you Start
+
+This tutorial describes how to implement the native [Sign In With Apple](https://developer.apple.com/sign-in-with-apple/) introduced in iOS 13. If instead you would like to implement Web Authentication with Auth0, please check out the [iOS Swift Login tutorial](/quickstart/native/ios-swift).
```

Comment:
I'd go with "Alternatively check out the [iOS Swift Login tutorial](/quickstart/native/ios-swift) quickstart for Web Authentication with Auth0."  Less wordy.